mobile/graphics.html

```diff
@@ -41,6 +41,10 @@ <h2>Technologies in progress</h2>
           <p>Animations can also be managed via scripting through the API exposed in <a data-featureid="web-animations">Web Animations</a>.</p>
         </div>
 
+        <div data-feature="2D Programmatic API">
+          <p><a data-featureid="css-paint-api">CSS Painting API</a> allows web developers to generate a 2D image used in CSS using JavaScript.</p>
```

Comment:
I would complete the description (it's not only about generating, it's also about repainting in response to style and size changes) and explain the benefits compared to doing the work purely in JavaScript:

```html
<p><a data-featureid="css-paint-api">CSS Painting API</a> allows web developers to generate a 2D image for CSS that can respond to style and size changes, without having to create DOM elements and without blocking the main thread during repaint, thus preserving memory, CPU, and responsiveness.</p>
```